app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-        <h1> Here is the index page of your posts for a given user</h1>
\ No newline at end of file
+<div class='post-div'>
+    <div class="show-container"> 
+         <img src=<%=@posts.photo%> alt='Profile' class = 'cover-img'  />
+        <div>
+           <p> <%= @posts.name %></h3></p>
+           <p>Number of posts: <%= @posts.posts_counter %> </p>
+        </div>
+    </div>
```

Comment:
- [ ] Good job making the requested changes however a routing error has been introduced that is blocking the user from going to the index page for a post, please fix this

Before clicking on the post
![image](https://github.com/techmoves/movesBloc/assets/81932478/660ca809-2c84-47b4-b84a-fa6ead8d0da0)

After
![image](https://github.com/techmoves/movesBloc/assets/81932478/a90ba9aa-f5cf-441d-ad5e-cfbe260f9d9d)